### DISCUSSION TITLE
How do I add multiple *arr tags to a collection?

### DISCUSSION THREAD

**Discussion by chris6611** [meisnate12/Plex-Meta-Manager#826] [General]
Hey guys, I have tried a couple different ways to add multiple tags.

I first used just pmm as a tag which worked just fine, but then I decided I wanted tags for each collection to identify them separately, so I changed my radarr_tag line to `radarr_tag: pmm, hulu` and that didn't work, only pmm was getting added still. I also tried `radarr_tag: [pmm, hulu]` and this didn't work either. Does anyone know the specific formatting needed for pmm to apply multiple tags, and if you change the radarr_tag parameter do the new tags apply retroactively to movies already in that connection without the tag?

**Comment by meisnate12** [comment]
try:
```yaml
radarr_tag: ["pmm", "hulu"]
```
or 
```yaml
radarr_tag:
  - pmm
  - hulu
```

**Reply by chris6611** [reply]
Quotes worked, thanks for the suggestion! By the way do you have any idea about what's going on with the overlay bug I posted about yesterday?

**Reply by meisnate12** [reply]
im going to be honest i completely ignored that because im almost done completely reworking overlays for 1.17.0 and i dont wanna spend time trying to fix something im going to toss shortly
